packages/browser-sdk/src/toolbar/Toolbar.tsx

```diff
@@ -43,7 +43,8 @@ export default function Toolbar({
   const [flags, setFlags] = useState<Flag[]>([]);
 
   const wasHidden =
-    window?.sessionStorage?.getItem(TOOLBAR_HIDE_KEY) === "true";
+    typeof window !== "undefined" &&
+    window.sessionStorage?.getItem(TOOLBAR_HIDE_KEY) === "true";
```

Comment:
since we have a few places where we access local storage is it worth factoring this access out so you only need this "window" logic in once place?

Reply:
ah, OK. Nevermind then 👍